### DISCUSSION TITLE
Welcome to Lightning Discussions!

### DISCUSSION THREAD

**Discussion by williamFalcon** [Lightning-AI/pytorch-lightning#5246] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Borda** [comment]
## Contributors

These stats are generated with [repoStats](https://borda.github.io/pyRepoStats/).
We have filtered users with 2 or more issues/PRs...

| user                                                                  |   merged PRs |   commented PRs |   opened issues |   commented issues |
|:----------------------------------------------------------------------|-------------:|----------------:|----------------:|-------------------:|
| [Borda](https://github.com/Borda)                                     |          715 |            1444 |              87 |                810 |
| [williamFalcon](https://github.com/williamFalcon)                     |          554 |             501 |             114 |                717 |
| [awaelchli](https://github.com/awaelchli)                             |          518 |            1382 |             111 |               1032 |
| [carmocca](https://github.com/carmocca)                               |          487 |            1067 |              87 |                480 |
| [rohitgr7](https://github.com/rohitgr7)                               |          235 |             560 |              58 |                507 |
| [tchaton](https://github.com/tchaton)                                 |          217 |             740 |              85 |                628 |
| [kaushikb11](https://github.com/kaushikb11)                           |          205 |             277 |              36 |                118 |
| [ananthsub](https://github.com/ananthsub)                             |          146 |             534 |             109 |                274 |
| [SeanNaren](https://github.com/SeanNaren)                             |          137 |             188 |              29 |                112 |
| [daniellepintz](https://github.com/daniellepintz)                     |           79 |              67 |              61 |                 36 |
| [SkafteNicki](https://github.com/SkafteNicki)                         |           72 |             174 |               4 |                206 |
| [akihironitta](https://github.com/akihironitta)                       |           65 |             136 |              49 |                 75 |
| [justusschock](https://github.com/justusschock)                       |           58 |             429 |              22 |                206 |
| [ethanwharris](https://github.com/ethanwharris)                       |           49 |              94 |              14 |                 46 |
| [edenlightning](https://github.com/edenlightning)                     |           48 |              93 |              89 |                311 |
| [ydcjeff](https://github.com/ydcjeff)                                 |           44 |              52 |               6 |                 83 |
| [four4fish](https://github.com/four4fish)                             |           42 |              90 |              33 |                 15 |
| [jjenniferdai](https://github.com/jjenniferdai)                       |           24 |               6 |               4 |                  7 |
| [mauvilsa](https://github.com/mauvilsa)                               |           23 |              11 |               4 |                 35 |
| [jeremyjordan](https://github.com/jeremyjordan)                       |           22 |              55 |              18 |                 44 |
| [neggert](https://github.com/neggert)                                 |           20 |              32 |              12 |                 50 |
| [arnaudgelas](https://github.com/arnaudgelas)                         |           20 |               0 |               4 |                  1 |
| [DuYicong515](https://github.com/DuYicong515)                         |           20 |               6 |               8 |                  1 |
| [s-rog](https://github.com/s-rog)                                     |           19 |              59 |              21 |                 62 |
| [lezwon](https://github.com/lezwon)                                   |           17 |              13 |               7 |                 33 |
| [nateraw](https://github.com/nateraw)                                 |           17 |              14 |               5 |                 32 |
| [ananyahjha93](https://github.com/ananyahjha93)                       |           16 |              22 |               6 |                 66 |
| [teddykoker](https://github.com/teddykoker)                           |           16 |              26 |               3 |                 27 |
| [deepsource-autofix[bot]](https://github.com/deepsource-autofix[bot]) |           16 |               0 |               0 |                  0 |
| [borisdayma](https://github.com/borisdayma)                           |           15 |               5 |               4 |                 36 |
| [mathemusician](https://github.com/mathemusician)                     |           15 |               2 |               0 |                  2 |
| [Ir1d](https://github.com/Ir1d)                                       |           14 |               7 |              11 |                 21 |
| [yukw777](https://github.com/yukw777)                                 |           14 |              13 |               8 |                 22 |
| [kuynzereb](https://github.com/kuynzereb)                             |           13 |               5 |               2 |                  6 |
| [stancld](https://github.com/stancld)                                 |           13 |               0 |               1 |                  1 |
| [akashkw](https://github.com/akashkw)                                 |           12 |               1 |               2 |                  3 |
| [tullie](https://github.com/tullie)                                   |           11 |              23 |               3 |                 12 |
| [ddrevicky](https://github.com/ddrevicky)                             |           10 |               1 |               2 |                  9 |
| [bryant1410](https://github.com/bryant1410)                           |           10 |               0 |               4 |                  4 |
| [Tshimanga](https://github.com/Tshimanga)                             |           10 |               2 |               0 |                  3 |
| [bamblebam](https://github.com/bamblebam)                             |           10 |               0 |               0 |                  8 |
| [zcain117](https://github.com/zcain117)                               |            9 |               7 |               5 |                  2 |
| [shuyingsunshine21](https://github.com/shuyingsunshine21)             |            9 |               2 |              11 |                  5 |
| [krshrimali](https://github.com/krshrimali)                           |            9 |              21 |               3 |                  5 |
| [tgaddair](https://github.com/tgaddair)                               |            8 |              22 |               1 |                  7 |
| [sebftw](https://github.com/sebftw)                                   |            7 |               0 |               4 |                  0 |
| [tadejsv](https://github.com/tadejsv)                                 |            7 |               4 |               6 |                  3 |
| [ninginthecloud](https://github.com/ninginthecloud)                   |            7 |               4 |               8 |                  7 |
| [siahuat0727](https://github.com/siahuat0727)                         |            6 |               0 |               2 |                  0 |
| [pre-commit-ci[bot]](https://github.com/pre-commit-ci[bot])           |            6 |               0 |               0 |                  0 |
| [yifuwang](https://github.com/yifuwang)                               |            6 |              16 |               7 |                 10 |
| [edward-io](https://github.com/edward-io)                             |            6 |               7 |               5 |                  3 |
| [puhuk](https://github.com/puhuk)                                     |            6 |               0 |               1 |                  5 |
| [kragniz](https://github.com/kragniz)                                 |            5 |               2 |               0 |                  0 |
| [bmartinn](https://github.com/bmartinn)                               |            5 |               3 |               1 |                  3 |
| [EricCousineau-TRI](https://github.com/EricCousineau-TRI)             |            5 |               1 |               5 |                  0 |
| [Vozf](https://github.com/Vozf)                                       |            5 |               2 |              14 |                  4 |
| [f4hy](https://github.com/f4hy)                                       |            5 |               4 |               1 |                  3 |
| [EspenHa](https://github.com/EspenHa)                                 |            5 |               1 |               4 |                  4 |
| [eladsegal](https://github.com/eladsegal)                             |            5 |               1 |              11 |                  2 |
| [romesco](https://github.com/romesco)                                 |            5 |               3 |               3 |                  5 |
| [kandluis](https://github.com/kandluis)                               |            5 |              11 |               3 |                  5 |
| [festeh](https://github.com/festeh)                                   |            4 |               3 |               5 |                 15 |
| [skmatz](https://github.com/skmatz)                                   |            4 |               0 |               0 |                  1 |
| [cmpute](https://github.com/cmpute)                                   |            4 |               0 |               6 |                  4 |
| [areshytko](https://github.com/areshytko)                             |            4 |               0 |               6 |                  0 |
| [Raalsky](https://github.com/Raalsky)                                 |            4 |               1 |               0 |                  0 |
| [tejasvi](https://github.com/tejasvi)                                 |            4 |               0 |               0 |                  1 |
| [kingyiusuen](https://github.com/kingyiusuen)                         |            4 |               0 |               0 |                  6 |
| [Rikorose](https://github.com/Rikorose)                               |            4 |               0 |               3 |                  0 |
| [cuent](https://github.com/cuent)                                     |            4 |               1 |               0 |                  3 |
| [olineumann](https://github.com/olineumann)                           |            4 |               1 |               2 |                 10 |
| [AndresAlgaba](https://github.com/AndresAlgaba)                       |            4 |               0 |               2 |                  3 |
| [ajtritt](https://github.com/ajtritt)                                 |            4 |               1 |               3 |                  1 |
| [Lucklyric](https://github.com/Lucklyric)                             |            4 |               0 |               1 |                  6 |
| [AlKun25](https://github.com/AlKun25)                                 |            4 |               0 |               0 |                  1 |
| [chunyang-wen](https://github.com/chunyang-wen)                       |            4 |               0 |               1 |                  0 |
| [wayi1](https://github.com/wayi1)                                     |            4 |               2 |               0 |                  1 |
| [jbschiratti](https://github.com/jbschiratti)                         |            4 |               0 |               1 |                  3 |
| [akshaykvnit](https://github.com/akshaykvnit)                         |            4 |               0 |               2 |                  1 |
| [krishnakalyan3](https://github.com/krishnakalyan3)                   |            4 |               0 |               0 |                  4 |
| [pbmstrk](https://github.com/pbmstrk)                                 |            4 |               0 |               8 |                  4 |
| [S-aiueo32](https://github.com/S-aiueo32)                             |            4 |               1 |              11 |                  3 |
| [sid-sundrani](https://github.com/sid-sundrani)                       |            4 |               0 |               7 |                  1 |
| [alanhdu](https://github.com/alanhdu)                                 |            4 |               0 |              17 |                  2 |
| [tarepan](https://github.com/tarepan)                                 |            4 |               1 |              11 |                  2 |
| [ruotianluo](https://github.com/ruotianluo)                           |            4 |               2 |              23 |                  2 |
| [alexeykarnachev](https://github.com/alexeykarnachev)                 |            4 |               2 |               6 |                  4 |
| [EricWiener](https://github.com/EricWiener)                           |            3 |               0 |              11 |                  1 |
| [ryanking13](https://github.com/ryanking13)                           |            3 |               0 |               2 |                  0 |
| [shabie](https://github.com/shabie)                                   |            3 |               0 |               1 |                  6 |
| [ceyzaguirre4](https://github.com/ceyzaguirre4)                       |            3 |               0 |               3 |                  0 |
| [nrupatunga](https://github.com/nrupatunga)                           |            3 |               1 |               1 |                  0 |
| [marload](https://github.com/marload)                                 |            3 |               4 |               3 |                  2 |
| [amogkam](https://github.com/amogkam)                                 |            3 |               0 |               4 |                  1 |
| [david-waterworth](https://github.com/david-waterworth)               |            3 |               0 |              16 |                  2 |
| [moi90](https://github.com/moi90)                                     |            3 |               1 |               5 |                  2 |
| [alok](https://github.com/alok)                                       |            3 |               1 |               4 |                  2 |
| [PhilJd](https://github.com/PhilJd)                                   |            3 |               0 |               3 |                  0 |
| [cowwoc](https://github.com/cowwoc)                                   |            3 |               6 |              13 |                 14 |
| [djbyrne](https://github.com/djbyrne)                                 |            3 |               0 |               2 |                  5 |
| [davinnovation](https://github.com/davinnovation)                     |            3 |               0 |              10 |                  2 |
| [rasbt](https://github.com/rasbt)                                     |            3 |               0 |               4 |                  1 |
| [brettkoonce](https://github.com/brettkoonce)                         |            3 |               0 |               0 |                  0 |
| [uditarora](https://github.com/uditarora)                             |            3 |               0 |               2 |                  1 |
| [authman](https://github.com/authman)                                 |            3 |               0 |               2 |                  1 |
| [Lucas-Steinmann](https://github.com/Lucas-Steinmann)                 |            3 |               0 |               3 |                  1 |
| [speediedan](https://github.com/speediedan)                           |            3 |               0 |               3 |                  0 |
| [NumesSanguis](https://github.com/NumesSanguis)                       |            3 |               3 |              16 |                  3 |
| [EliaCereda](https://github.com/EliaCereda)                           |            3 |               0 |               2 |                  1 |
| [c00k1ez](https://github.com/c00k1ez)                                 |            3 |               0 |               2 |                  2 |
| [kepler](https://github.com/kepler)                                   |            3 |               0 |               0 |                  1 |
| [thschaaf](https://github.com/thschaaf)                               |            3 |               1 |               3 |                  0 |
| [sumanthratna](https://github.com/sumanthratna)                       |            3 |               0 |               1 |                  0 |
| [hadim](https://github.com/hadim)                                     |            3 |              11 |               6 |                 12 |
| [maxjeblick](https://github.com/maxjeblick)                           |            3 |               0 |               7 |                  3 |
| [cinjon](https://github.com/cinjon)                                   |            3 |               0 |               1 |                  1 |
| [ibeltagy](https://github.com/ibeltagy)                               |            3 |               2 |               9 |                  2 |
| [mpariente](https://github.com/mpariente)                             |            3 |               2 |               6 |                  3 |
| [ozen](https://github.com/ozen)                                       |            3 |               2 |               5 |                  3 |
| [akhti](https://github.com/akhti)                                     |            3 |               1 |               0 |                  0 |
| [thomasjpfan](https://github.com/thomasjpfan)                         |            3 |               0 |               1 |                  0 |
| [manipopopo](https://github.com/manipopopo)                           |            3 |               0 |               6 |                  1 |
| [elliotwaite](https://github.com/elliotwaite)                         |            2 |               0 |               2 |                  0 |
| [zredeaux65](https://github.com/zredeaux65)                           |            2 |               0 |               0 |                  0 |
| [zippeurfou](https://github.com/zippeurfou)                           |            2 |               0 |               1 |                  2 |
| [sholalkere](https://github.com/sholalkere)                           |            2 |               0 |               2 |                  4 |
| [ybrovman](https://github.com/ybrovman)                               |            2 |               0 |               0 |                  0 |
| [heytitle](https://github.com/heytitle)                               |            2 |               0 |               3 |                  0 |
| [ivannz](https://github.com/ivannz)                                   |            2 |               1 |               1 |                  0 |
| [calebrob6](https://github.com/calebrob6)                             |            2 |               1 |               0 |                  2 |
| [roshikouhai](https://github.com/roshikouhai)                         |            2 |               0 |               0 |                  0 |
| [SpontaneousDuck](https://github.com/SpontaneousDuck)                 |            2 |               1 |               3 |                  1 |
| [ashleve](https://github.com/ashleve)                                 |            2 |               1 |               8 |                  0 |
| [dipam7](https://github.com/dipam7)                                   |            2 |               0 |               0 |                  2 |
| [jonashaag](https://github.com/jonashaag)                             |            2 |               0 |               4 |                  1 |
| [monney](https://github.com/monney)                                   |            2 |               0 |               4 |                  0 |
| [nsarang](https://github.com/nsarang)                                 |            2 |               0 |               0 |                  6 |
| [Lucien-cs](https://github.com/Lucien-cs)                             |            2 |               0 |               0 |                  0 |
| [ramonemiliani93](https://github.com/ramonemiliani93)                 |            2 |               0 |               4 |                  0 |
| [hemildesai](https://github.com/hemildesai)                           |            2 |               0 |               1 |                  2 |
| [janhenriklambrechts](https://github.com/janhenriklambrechts)         |            2 |               0 |               1 |                  5 |
| [tshrjn](https://github.com/tshrjn)                                   |            2 |               0 |               4 |                  2 |
| [ghost](https://github.com/ghost)                                     |            2 |               0 |              13 |                  5 |
| [dreamgonfly](https://github.com/dreamgonfly)                         |            2 |               0 |               4 |                  1 |
| [HarshSharma12](https://github.com/HarshSharma12)                     |            2 |               0 |               2 |                  1 |
| [JanSellner](https://github.com/JanSellner)                           |            2 |               0 |               2 |                  0 |
| [davors72](https://github.com/davors72)                               |            2 |               4 |               0 |                  1 |
| [gtauzin](https://github.com/gtauzin)                                 |            2 |               0 |               0 |                  0 |
| [THasthika](https://github.com/THasthika)                             |            2 |               0 |               2 |                  0 |
| [irene50288](https://github.com/irene50288)                           |            2 |               0 |               0 |                  0 |
| [popfido](https://github.com/popfido)                                 |            2 |               0 |               0 |                  2 |
| [zerogerc](https://github.com/zerogerc)                               |            2 |               0 |               0 |                  3 |
| [tangbinh](https://github.com/tangbinh)                               |            2 |               0 |               0 |                  3 |
| [shivdhar](https://github.com/shivdhar)                               |            2 |               0 |               1 |                  1 |
| [humandotlearning](https://github.com/humandotlearning)               |            2 |               0 |               0 |                  1 |
| [peteriz](https://github.com/peteriz)                                 |            2 |               0 |               2 |                  1 |
| [asafmanor](https://github.com/asafmanor)                             |            2 |               0 |               0 |                  1 |
| [edgarriba](https://github.com/edgarriba)                             |            2 |               6 |               0 |                 13 |
| [psinger](https://github.com/psinger)                                 |            2 |               0 |               1 |                  1 |
| [ericharper](https://github.com/ericharper)                           |            2 |               0 |               3 |                  2 |
| [ylsung](https://github.com/ylsung)                                   |            2 |               1 |               3 |                  0 |
| [george-gca](https://github.com/george-gca)                           |            2 |               1 |               3 |                  6 |
| [alessiobonfiglio](https://github.com/alessiobonfiglio)               |            2 |               0 |               2 |                  0 |
| [8greg8](https://github.com/8greg8)                                   |            2 |               0 |               3 |                  0 |
| [rjkilpatrick](https://github.com/rjkilpatrick)                       |            2 |               0 |               0 |                  0 |
| [kevinc13](https://github.com/kevinc13)                               |            2 |               0 |               2 |                  1 |
| [jtamir](https://github.com/jtamir)                                   |            2 |               0 |               0 |                  0 |
| [rzepinskip](https://github.com/rzepinskip)                           |            2 |               0 |               3 |                  1 |
| [asrafulashiq](https://github.com/asrafulashiq)                       |            2 |               1 |               3 |                  2 |
| [SauravMaheshkar](https://github.com/SauravMaheshkar)                 |            2 |               0 |               0 |                  1 |
| [abhinavg97](https://github.com/abhinavg97)                           |            2 |               0 |               2 |                  0 |
| [ibraheem-moosa](https://github.com/ibraheem-moosa)                   |            2 |               0 |               0 |                  0 |
| [nanddalal](https://github.com/nanddalal)                             |            2 |               0 |               0 |                  0 |
| [jakubczakon](https://github.com/jakubczakon)                         |            2 |               1 |               3 |                  5 |
| [xerus](https://github.com/xerus)                                     |            2 |               0 |               1 |                  0 |
| [aiyolo](https://github.com/aiyolo)                                   |            2 |               0 |               8 |                  5 |
| [maximsch2](https://github.com/maximsch2)                             |            2 |               4 |               1 |                  2 |
| [adamreeve](https://github.com/adamreeve)                             |            2 |               0 |               0 |                  1 |
| [catalys1](https://github.com/catalys1)                               |            2 |               0 |               6 |                  2 |
| [jiwidi](https://github.com/jiwidi)                                   |            2 |               0 |               6 |                  1 |
| [manangoel99](https://github.com/manangoel99)                         |            2 |               1 |               0 |                  4 |
| [neighthan](https://github.com/neighthan)                             |            2 |               0 |               4 |                  0 |
| [deng-cy](https://github.com/deng-cy)                                 |            2 |               1 |               5 |                  3 |
| [nathanpainchaud](https://github.com/nathanpainchaud)                 |            2 |               1 |               9 |                  2 |
| [Diuven](https://github.com/Diuven)                                   |            2 |               0 |               1 |                  1 |
| [aribornstein](https://github.com/aribornstein)                       |            2 |               3 |               5 |                  6 |
| [krfricke](https://github.com/krfricke)                               |            2 |               0 |               0 |                  0 |
| [InCogNiTo124](https://github.com/InCogNiTo124)                       |            2 |               0 |               1 |                  6 |
| [twsl](https://github.com/twsl)                                       |            2 |               0 |               5 |                  7 |
| [wangraying](https://github.com/wangraying)                           |            2 |               0 |               0 |                  0 |
| [gaoteng-git](https://github.com/gaoteng-git)                         |            2 |               0 |               1 |                  1 |
| [shreyasbapat](https://github.com/shreyasbapat)                       |            2 |               0 |               1 |                  2 |
| [bilal2vec](https://github.com/bilal2vec)                             |            2 |               0 |               5 |                  1 |
| [kartik4949](https://github.com/kartik4949)                           |            2 |               0 |               4 |                  0 |
| [DKandrew](https://github.com/DKandrew)                               |            2 |               0 |               6 |                  3 |
| [MattPainter01](https://github.com/MattPainter01)                     |            2 |               5 |               1 |                  2 |
| [creafz](https://github.com/creafz)                                   |            2 |               0 |               1 |                  0 |
| [ashwinb](https://github.com/ashwinb)                                 |            2 |               1 |               1 |                  0 |
| [quancs](https://github.com/quancs)                                   |            2 |               0 |               7 |                  3 |
| [NathanGodey](https://github.com/NathanGodey)                         |            2 |               0 |               1 |                  0 |
| [anthonytec2](https://github.com/anthonytec2)                         |            2 |               1 |               6 |                  1 |
| [srush](https://github.com/srush)                                     |            2 |               2 |               5 |                  4 |
| [jerome-habana](https://github.com/jerome-habana)                     |            2 |               0 |               0 |                  0 |
| [gianscarpe](https://github.com/gianscarpe)                           |            2 |               3 |               6 |                  4 |
| [leezu](https://github.com/leezu)                                     |            2 |               6 |               8 |                  3 |
| [jeffling](https://github.com/jeffling)                               |            2 |              20 |               1 |                 28 |
| [guyang3532](https://github.com/guyang3532)                           |            2 |               1 |               1 |                  0 |
| [LaserBit](https://github.com/LaserBit)                               |            2 |               0 |               1 |                  0 |
| [tshu-w](https://github.com/tshu-w)                                   |            2 |               3 |               8 |                  4 |
| [borchero](https://github.com/borchero)                               |            2 |               0 |               1 |                  0 |
| [mateuszpieniak](https://github.com/mateuszpieniak)                   |            2 |               0 |               5 |                  5 |
| [TheStoneMX](https://github.com/TheStoneMX)                           |            2 |               0 |               1 |                  1 |
| [ayberkydn](https://github.com/ayberkydn)                             |            2 |               0 |               2 |                  2 |
| [david-codaio](https://github.com/david-codaio)                       |            2 |               0 |               4 |                  0 |
| [matthew-z](https://github.com/matthew-z)                             |            2 |               0 |               3 |                  5 |
| [kumuji](https://github.com/kumuji)                                   |            2 |               0 |               1 |                  1 |
| [qqueing](https://github.com/qqueing)                                 |            2 |               0 |               4 |                  1 |
| [rmrao](https://github.com/rmrao)                                     |            2 |               0 |               4 |                  3 |
| [zphang](https://github.com/zphang)                                   |            2 |               0 |               1 |                  1 |
| [expectopatronum](https://github.com/expectopatronum)                 |            2 |               1 |               2 |                  6 |

**Reply by Borda** [reply]
![Figure_1](https://user-images.githubusercontent.com/6035284/147274754-097a523d-56ed-4bad-98ec-cf131ca9a434.png)

**Comment by VisionH222** [comment]
I account a problem. 
5 D:\Anaconda\envs\VH\lib\site-packages\pytorch_lightning\utilities\parsing.py:104: UserWarning: attribute 'model' removed from hparams because it cannot be pickled
6   rank_zero_warn(f"attribute '{k}' removed from hparams because it cannot be pickled")
7 D:\Anaconda\envs\VH\lib\site-packages\pytorch_lightning\utilities\parsing.py:104: UserWarning: attribute 'loss_fn' removed from hparams because it cannot be pickled
8   rank_zero_warn(f"attribute '{k}' removed from hparams because it cannot be pickled")
so I can't  use my checkpoint to load my model, can you help me?
